gnovm/tests/files/zrealm_crossrealm28_array.gno

```diff
@@ -0,0 +1,45 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	crossrealm "gno.land/r/demo/tests/crossrealm/array"
+)
+
+var root interface{}
+
+func main() {
+	root = crossrealm.GetArray5()
+	println(".")
+}
+
+// Output:
+// .
+
```

Comment:
this works even the result of `GetArray5()` is already real. the value is copied to current realm, and it's not bound to external realm, because its Type is not a declared type bound to the external realm.